app.js

```diff
@@ -26,8 +26,12 @@
  */
 'use strict';
 
-var Interpreter = require('./lib/interpreter.js');
-var cli = new Interpreter();
-cli.supressWarmupMessages = true;
-cli.startup();
-cli.handle(process.argv, true);
+var updateCheck = require('./lib/update-check');
+
+updateCheck(function () {
+	var Interpreter = require('./lib/interpreter.js');
+	var cli = new Interpreter();
+	cli.supressWarmupMessages = true;
+	cli.startup();
+	cli.handle(process.argv, true);
+});
```

Comment:
Any way to run the update code *after* the CLI command? That would create less disruption. It would also be possible to Ctrl-C the update check without issue.